src/java.base/unix/native/libjava/java_props_md.c

```diff
@@ -388,8 +388,12 @@ GetJavaProperties(JNIEnv *env)
     /* supported instruction sets */
     {
         char list[258];
-        sysinfo(SI_ISALIST, list, sizeof(list));
-        sprops.cpu_isalist = strdup(list);
+        int ret = sysinfo(SI_ISALIST, list, sizeof(list));
+        if (ret == 0) {
+            sprops.cpu_isalist = strdup(list);
+        } else {
+            sprops.cpu_isalist = NULL;
```

Comment:
Would you mind checking the code that puts this in a system property can tolerate NULL? I don't suppose sysinfo will fail here (unless you found a case?) but it would be good to confirm that it won't blow up somewhere else.

Reply:
We do  PUTPROP here
https://github.com/openjdk/jdk/blob/5d65c23cd99b72527dcfab9eb6da9510e7dc6330/src/java.base/share/native/libjava/System.c#L164
with 3rd parameter == NULL in this case.
PUTPROP checks for this so I do not see issues with this
https://github.com/openjdk/jdk/blob/5d65c23cd99b72527dcfab9eb6da9510e7dc6330/src/java.base/share/native/libjava/System.c#L68